Profiler/log/TraceLog.cpp

```diff
@@ -0,0 +1,58 @@
+#include "TraceLog.h"
+#include "version.h"
+#include <vector>
+#include <fstream>
+#include <algorithm>
+#include <winuser.h>
+#include "utils/WindowsUtils.h"
+#include <string>
+
+
+void TraceLog::writeJittedFunctionInfosToLog(std::vector<FunctionInfo>* functions)
+{
+	writeFunctionInfosToLog(LOG_KEY_JITTED, functions);
+}
+
+void TraceLog::writeInlinedFunctionInfosToLog(std::vector<FunctionInfo>* functions)
+{
+	writeFunctionInfosToLog(LOG_KEY_INLINED, functions);
+}
+
+void TraceLog::createLogFile(Config& config) {
+	std::string targetDir = config.getTargetDir();
+
+	char timeStamp[BUFFER_SIZE];
+	getFormattedCurrentTime(timeStamp, sizeof(timeStamp));
```

Comment:
Here you have the logic for getting a timestamp string again (as in the `AttachLog`), but you don't need to convert it to a string before concatenating it into the log message. Why the difference? Can the same logic/helper method be used in all three places?